demo-app/src/main/kotlin/io/getstream/video/android/ui/login/LoginViewModel.kt

```diff
@@ -20,14 +20,14 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import dagger.hilt.android.lifecycle.HiltViewModel
 import io.getstream.log.streamLog
-import io.getstream.video.android.API_KEY
```

Comment:
If API_KEY is not used anymore you can also remove some last traces of it. In `.env.properties.sample` you can delete:

```
DOGFOODING_BUILD_CONFIG_API_KEY=
PRODUCTION_BUILD_CONFIG_API_KEY=
```

And then in your local `.env.properties` also delete these and try if it builds.

Reply:
Done. Also I'll update the docs.